qt/Torrent.h

```diff
@@ -590,33 +509,78 @@ class Torrent : public QObject
     static KeyList const mainStatKeys;
 
 private:
-    int getInt(int key) const;
-    bool getBool(int key) const;
-    QIcon getIcon(int key) const;
-    double getDouble(int key) const;
-    qulonglong getSize(int key) const;
-    QString getString(int key) const;
-    time_t getTime(int key) const;
-
-    bool setInt(int key, int value);
-    bool setBool(int key, bool value);
-    bool setIcon(int key, QIcon const&);
-    bool setDouble(int key, double);
-    bool setString(int key, char const*, size_t len);
-    bool setSize(int key, qulonglong);
-    bool setTime(int key, time_t);
+    void updateMimeIcon();
+
+private:
+    int const id_;
```

Comment:
Is this the new member variables naming convention? I'd like it to be unified one day...

Reply:
Hmm, this one's a little tricky. I wasn't intending to set precedent here; the field names in Torrent are a special case driven by the code's needs:

 I used `foo_` here rather than `myFoo` where `foo` is one of the tr_quarks s.t. it's amenable to C preprocessor token pasting, e.g. 

```cpp
  #define HANDLE_KEY(key) case TR_KEY_ ## key: \
      field_changed |= change(key ## _, child); break;
```

...using the `myFoo` form here along with token pasting would lose the capitalization in the variable name, e.g. `myfoo`. This is ugly, so I went with the trailing underscore instead.

As for whether to use this as a new convention replacing `myFoo` in the Qt client --  that wasn't something I'd considered, but TBH I'd have a *slight* preference for that and will make a followup PR to do that if you also like the idea.

Reply:
I don't have a preference here (and Transmission is the first and only project where I've seen this `myXxx` convention; FWIW I'm usually using `m_xxx`) except for consistency. Would welcome a PR or could do it myself, as I understand it may be a bit time-consuming.

Reply:
Cool, we'll dump the `myXxx` notation in a followup.

My vote would be for `xxx_`, but it's not a strong preference and I'd be perfectly fine with `m_xxx` too.

Since https://github.com/transmission/transmission/pull/1236 touches a lot of `qt/` files, it would be simpler to land 1234 and 1236 first... :slightly_smiling_face: